config/config.go

```diff
@@ -203,13 +202,17 @@ func Get(cf string, ec *Extended, logger zap.Logger) (*AppConfig, error) {
 
 	cfg := AppConfig{
 		Logging:                b.Logging,
-		Verbose:                b.Verbose,
 		PIDPath:                b.Arachne.PIDPath,
 		Orchestrator:           b.Arachne.Orchestrator,
 		StandaloneTargetConfig: b.Arachne.StandaloneTargetConfig,
 		Metrics:                mc,
 	}
 
+	if !cfg.Orchestrator.Enabled && cfg.StandaloneTargetConfig == "" {
```

Comment:
Interesting, it looks like we can potentially silence an error above on L197 (I just now noticed); I can't comment on that line, because github, but noticed when trying to remind myself what `err` is here.

Right, okay so that said then, `err` is always nil at this point, or we would've returned on line 200.

Can we just return an `errors.New("the standalone...")` rather than logging and returning `nil, nil`?